mesonbuild/backend/ninjabackend.py

```diff
@@ -1808,7 +1808,7 @@ def generate_genlist_for_target(self, genlist, target, outfile):
                 elem.add_item('DEPFILE', depfile)
             if len(extra_dependencies) > 0:
                 elem.add_dep(extra_dependencies)
-            elem.add_item('DESC', 'Generating $out')
+            elem.add_item('DESC', 'Generating \'%s\'.' % sole_output)
```

Comment:
This will break other things that use `DESC` in `CUSTOM_COMMAND`, such as the `benchmark` target. I think the simplest solution is to add `elem.add_item_noescape()` and use it for descriptions.